openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
Nit: we use JSDoc-style comments (http://usejsdoc.org/about-getting-started.html), so this should be:

```
/**
 * Enable users to switch between viewing and writing CourseTalk reviews.
 */
```